src/processing/farger/farger.md

```diff
@@ -0,0 +1,553 @@
+
+    Nå får du en mørkegul sirkel midt i vinduet på en blå bakgrunn.
+
++ Endre fargen på omrisset med funksjonen `stroke`:
+
+    ```processing
+    void draw() {
+      background(0, 0, 255);
+      fill(255, 192, 64);
+      stroke(192, 96, 64);
+      ellipse(width / 2, height / 2, 100, 100);
+    }
+    ```
+
+    Nå er streken rundt sirkelen en rødlig brun. Det er kanskje ikke
+    så lett å se fargen på streken siden den er så tynn.
+
++ Gjør omrisset fetere med funksjonen `strokeWeight`:
+
+    ```processing
+    void draw() {
+      background(0, 0, 255);
+      fill(255, 192, 64);
+      stroke(192, 96, 64);
+      strokeWeight(3);
+      ellipse(width / 2, height / 2, 100, 100);
+    }
+    ```
+
+    Nå er omrisset tre piksler bredt.
+
+## Eksperimenter {.try}
+
++ Prøv forskjellige bakgrunnsfarger. Hvordan synes du forskjellige
+  bakgrunnsfarger passer sammen med fargene på sirkelen?
++ Prøv forskjellige fyllfarger. Hvordan passer disse med fargen på
+  omrisset?
++ Prøv forskjellige farger på omrisset. Hvordan passer det med
+  bakgrunnen og fyllfargen?
++ Prøv andre tykkelser på omrisset. Hvor synes du at det er passe
+  tykt?
++ Kan du tegne to sirkler på skjermen i forskjellige farger?
+
+# Steg 3: Fargevelgeren {.activity}
+
+Noen ganger kan det være tungvint å skulle lage fargene man har lyst
+på ved å bare tenke ut tallene. Man blir riktignok flinkere på dette
+etter hvert som man har gjort det en del ganger. Inntil man har blitt
+komfortabel med dette, kan det være lurt å bruke fargevelgeren som
+finnes i Processing.
+
+## Sjekkliste {.check}
+
++ Åpne fargevelgeren ved å velge **Tools → Color Selector**
+
+    ![](Fargevelger.png "Fargevelgeren slik den er når den først åpnes.")
+
+    Du får da opp et vindu som lar deg velge farger. Her kan du fylle
+    inn tallverdiene som vi har sett tidligere, R, G og B. Det er også
+    noen andre felter hvor du kan fylle inn verdier. Det ene er for et
+    annet fargesystem enn RGB. Det er også noen områder som har farge.
+
++ Prøv å trykke i fargefeltet til venstre. Her kan du velge hvor sterk
+  og hvor lys fargen skal være.
+
+    Hva skjer med verdiene i de forskjellige tekstfeltene? Kan du se
+    hvordan de to tekstfeltene S og B henger sammen hvilken farge du
+    velger i dette fargefeltet?
+
++ Prøv å trykke i fargefeltet rett ved siden av. Her kan du velge
+  hvilken farge du vil ha.
+
+    ![](Fargevelger.gif "En animasjon som viser at fargen endres i
+     fargevelgeren.")
+
+    Du ser kanskje at øverst til høyre er det et fargefelt til. Her
+    vises den fargen du har valgt.
+
+    Hva skjer med verdiene i de forskjellige tekstfeltene nå? Hvordan
+    oppfører H, S og B seg sammenliknet med forrige gang?
+
++ Finn en farge du liker til bakgrunnsfarge. Skriv inn verdiene for R,
+  G og B inn i kallet på `background` i `draw`.
+
++ Finn en farge du liker til sirkelen. Trykk på knappen **Copy**. Visk
+  ut alle verdiene gitt i kallet på `fill` og lim inn den kopierte

```

Comment:
Burde vi unngå "kallet" på oppgaver med level 1? Kan kanskje skrives uten: Visk ut alle verdiene for 'fill'.